sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BulkExecutor.java

```diff
@@ -234,8 +299,12 @@ public BulkExecutor(CosmosAsyncContainer container,
                 return ((ItemBulkOperation<?>) itemOperation).getRetryPolicy().shouldRetry(operationResult).flatMap(
                     result -> {
                         if (result.shouldRetry) {
-                            return this.enqueueForRetry(result.backOffTime, groupSink, itemOperation);
+                            return this.enqueueForRetry(result.backOffTime, groupSink, itemOperation, thresholds);
                         } else {
+                            // Treat 408 RequestTimeout as being retried - need to minimize throughput
+                            if (operationResult.getStatusCode() == 408) {
```

Comment:
Just for personal understanding: 
so the 408 here is the one coming back from server?
The batch response is 207, but some of the operations failed with 408?

Reply:
Yes - but I actually retested this and the number of 408s isn't really influenced by this. Was a red herring. I have removed the 408 special-casing here.